test/handler.test.js

```diff
@@ -396,43 +396,96 @@ describe('strong-error-handler', function() {
 
       requestJson().expect(500).end(function(err, res) {
         if (err) return done(err);
-        expect(res.body).to.eql({
-          error: {
-            statusCode: 500,
-            message: 'Internal Server Error',
-          },
-        });
+        const data = res.body.error;
+        expect(data).to.have.property('message').that.match(/multiple errors/);
+        expect(data).to.have.property('details').eql([
+          {statusCode: 500, message: 'Internal Server Error'},
+          {statusCode: 500, message: 'Internal Server Error'},
+          {statusCode: 500, message: 'Internal Server Error'},
+        ]);
```

Comment:
This is a bit ugly. Before, we would return a short error response:

```js
{
  error: {
    statusCode: 500,
    message: 'Internal Server Error',
  },
});
```

Now we are returning more details, but these details are not really helpful:

```js
{
  error: {
    statusCode: 500,
    message: 'Internal Server Error',
    details: [
      {statusCode: 500, message: 'Internal Server Error'},
      {statusCode: 500, message: 'Internal Server Error'},
      {statusCode: 500, message: 'Internal Server Error'},
    ],
  }
}
```

It may be possible to implement some heuristics to omit these "generic" entries from the output, but don't think it's worth the effort and additional complexity.

Reply:
Agreed